graylog2-web-interface/src/views/hooks/SyncWithQueryParameters.js

```diff
@@ -31,9 +31,9 @@ const extractTimerangeParams = (timerange: TimeRange) => {
 };
 
 export const syncWithQueryParameters = (query: string, action: (string) => mixed = history.push) => {
-  const { view } = ViewStore.getInitialState() || {};
+  const { view, dirty } = ViewStore.getInitialState() || {};
```

Comment:
I do not think there is a way for `ViewStore.getInitialState()` to return falsy, so we may not need the fallback `|| {}`

```suggestion
  const { view, dirty } = ViewStore.getInitialState();
```